### ISSUE TITLE
zowe-verify-post-install.sh should determine ZOWESVR name itself

### ISSUE THREAD

**Issue by John-A-Davies** [zowe/zowe-install-packaging#168]
Currently the script checks the setting of ${ZOWESVR} for this value and if it's not set, it uses "ZOWESVR".
This script should instead discover the ZOWESVR name itself, e.g. from `$ZOWE_ROOT_DIR/scripts/zowe-start.sh`

**Comment by John-A-Davies** [comment]
Solution:  The script extracts the <procserver> name from `zowe-start.sh`.  Then it looks for jobs that are running the $ZOWE_ROOT_DIR code.  If the server name in `zowe-start.sh` does not match what's running that installed executable code, you will get this message:
```
Warning: Found PROC <procserver> in zowe-start.sh, but $ZOWE_ROOT_DIR code is running in <runserver> instead
Info: Switching to job <runserver>
```
The script opts for the `<runserver>` because the user might have used the z/OS START command rather than `zowe-start.sh`.  
The script also warns you if more than one job is running the $ZOWE_ROOT_DIR code and prints that list.  If none of those jobs match what it found in `zowe-start.sh`, it uses ZOWESVR.

**Comment by John-A-Davies** [comment]
Also fixes #175 by adding the missing redirection character.
